src/schema/slack/functions/_scripts/src/deps.ts

```diff
@@ -0,0 +1 @@
+export { pascalCase } from "https://deno.land/x/case@v2.1.0/mod.ts";
```

Comment:
Two questions
- Is there a reason we are giving this a separate `deps.ts` file instead of using our existing `src/deps.ts`?
- Should this be in a `dev_deps.ts` file since we only use it in this generation script?

Reply:
I was unsure where to put this, should have asked in the PR

Thanks for clearing it up 💯  I think `dev_deps.ts` is the best place to put this since it won't be required at runtime